src/components/Topics.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from "react";
+import { getTopics } from "../utils/api";
+import { Link } from "react-router-dom";
+
+export default function Topics() {
+  const [topic, settopics] = useState([]);
+  
+
+  useEffect(() => {
+    getTopics().then((res) => {
+      settopics(res);
+    });
+  }, [topic]);
+
```

Comment:
Brilliant that you're using an API call to populate the topics, rather than hard coding 👍